chemicaltoolbox/autodock_vina/prepare_ligand/.shed.yml

```diff
@@ -3,6 +3,9 @@ categories:
 description: Software for protein-compound docking - the prepare ligand step
 long_description: |
     AutoDock Vina is an open-source program for doing molecular docking.
+    Home page: http://vina.scripps.edu/
+    License: Apache 2.0
```

Comment:
I like the idea of including the license here ... we should follow up on this and integrate this deeply into Galaxy at some point. Maybe not in the shed.yml, but in the tool XML file.

Reply:
Would that be the license of the Galaxy tool (i.e. XML files and eventual scripts)?

For the tool requirements we should be able to get this info from the conda packages or some registry.

Reply:
What we need for a long time, for people like Brad, for example, is an easy way to see if a tool in a given Galaxy instance is "academic free only" etc ...

So this would be the license of the underlying tool(s). We can not depend on conda only I think, that's why I think we should have a `licenses` tag in a Galaxy tool. But I agree it should be simple for tool devs to obtain this from conda or bio.tools.

Reply:
> We can not depend on conda only I think, that's why I think we should have a `licenses` tag in a Galaxy tool.

Is that because not all conda packages have this information (in case I'd argue it's better to fix them there) or because of the other dependency resolution mechanism that Galaxy supports?

> But I agree it should be simple for tool devs to obtain this from conda or bio.tools.

It would still be a massive duplication of information.

Reply:
> Is that because not all conda packages have this information (in case I'd argue it's better to fix them there) or because of the other dependency resolution mechanism that Galaxy supports?

the latter ...

> It would still be a massive duplication of information.

Yes, but hard to avoid. The other solution is get finally an xref attribute into the tool specification (it's on the tools-iuc todo list since last summer), so we reference bio.tools- and similar ids. With that we could automatically retrieve the License information ... but this all only works for online-Galaxy instances and not offline Galaxy's. (Which is not much different for the citations I assume.)

Reply:
> The other solution is get finally an xref attribute into the tool specification (it's on the tools-iuc todo list since last summer), so we reference bio.tools- and similar ids. With that we could automatically retrieve the License information ... but this all only works for online-Galaxy instances and not offline Galaxy's. (Which is not much different for the citations I assume.)

I'd prefer such a solution, which would potentially allow us to retrieve other useful info from bio.tools .

Reply:
I think both are valid. I think IUC can decide which are mandatory in the end? bio.tools does not have all tools listed for example, at least if we talk about NLP, Astro or Climate tools ...